src/components/Composer/index.js

```diff
@@ -358,28 +398,52 @@ class Composer extends React.Component {
         propStyles.outline = 'none';
         const propsWithoutStyles = _.omit(this.props, 'style');
 
+        // This code creates a hidden text component that helps track the caret position in the visible input.
+
+        const calculationCaretPositionElement = (
+            <View
+                style={{
+                    position: 'absolute',
+                    bottom: -2000,
+                    zIndex: -1,
+                    opacity: 0,
+                }}
+            >
+                <Text
+                    multiline
+                    style={[styles.textInputCompose, {width: this.state.width}]}
+                >
+                    {this.state.valueBeforeCaret}
+                    <Text ref={this.textRef} />
+                </Text>
+            </View>
+        );
+
         // We're disabling autoCorrect for iOS Safari until Safari fixes this issue. See https://github.com/Expensify/App/issues/8592
         return (
-            <RNTextInput
-                autoComplete="off"
-                autoCorrect={!Browser.isMobileSafari()}
-                placeholderTextColor={themeColors.placeholderText}
-                ref={el => this.textInput = el}
-                selection={this.state.selection}
-                onChange={this.shouldCallUpdateNumberOfLines}
-                onSelectionChange={this.onSelectionChange}
```

Comment:
this is a question about old code, but they were we using `this.onSelectionChange`? This supposed to be `this.props.onSelectionChange` right?

Reply:
I understand your surprise, and you are correct. It should indeed be `this.props.onSelectionChange`